app/javascript/components/common/TrackIcon.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export function TrackIcon({ track }) {
+  return (
+    <img
+      style={{ width: 100 }}
```

Comment:
If this is a common component, it shouldn't have a width. I don't mind it going in temporarily, but could you add a `TODO` comment for it to be removed once the css class is implemented please.